spec/docs/early_career_teacher_participation_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "swagger_helper"
+
+RSpec.describe "Early Career Teacher Participation", type: :request do
+  let(:user) { create(:user) }
+  let(:lead_provider) { create(:lead_provider) }
+  let(:token) { LeadProviderApiToken.create_with_random_token!(lead_provider: lead_provider) }
+  let(:bearer_token) { "Bearer #{token}" }
+
+  before do
+    default_headers[:Authorization] = bearer_token
+  end
+
+  path "/early-career-teacher-participants" do
+    post "Add Early Career Teacher to Course" do
+      operationId :api_v1_create_ect_participant
+      tags "ect_participant"
+      consumes "application/json"
+      parameter name: "Authorization", in: :header, required: false, type: :string, description: "The bearer token associated with a lead provider"
+      parameter name: :params, in: :body, required: false, type: :string, description: "The unique id of the participant"
```

Comment:
Would this not be `in: :path` ?

Reply:
Also seems there is a validation issue with the generated swagger https://validator.swagger.io/validator/debug?url=https://ecf-review-pr-333.london.cloudapps.digital/api-docs/v1/api_spec.json

Reply:
> Would this not be `in: :path` ?

Needs to be in body. First pass thin slice is single participant, later we will expand to multiple.

Reply:
Now it's fixed -> https://ecf-review-pr-333.london.cloudapps.digital/api-docs
Split into public API doc and secure API doc, drop down list on top:

<img width="1450" alt="Screenshot 2021-05-11 at 16 01 38" src="https://user-images.githubusercontent.com/77098906/117838713-7abaa400-b272-11eb-902d-b5cf4773a71c.png">